configure.sh

```diff
@@ -29,7 +29,8 @@ The following flags enable optional features (disable with --no-<option name>).
   --optimized              optimize the build
   --debug-symbols          include debug symbols
   --valgrind               Valgrind instrumentation
-  --debug-context-mm       use the debug context memory manager
+  --debug-context-memory-manager
```

Comment:
This option name is very long, should we rename the option to `--debug-context-mm` and also rename all the other occurrences of it?